share/p2p/discovery/discovery.go

```diff
@@ -35,8 +35,8 @@ const (
 	retryTimeout = time.Second
 )
 
-// discoveryRetryTimeout defines time interval between discovery attempts, needed for tests
-var discoveryRetryTimeout = retryTimeout
+// DiscoveryRetryTimeout defines time interval between discovery attempts, needed for tests
+var DiscoveryRetryTimeout = time.Minute
```

Comment:
I start to feel a problem with that, like all the light nodes will have to wait for 1 min, after they start, which sucks. Maybe 10 secs?